### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,23 @@ Framework](https://dotnet.microsoft.com/download/dotnet-framework) except for
 `.NET Framework 3.5 SP1`. Any exceptions to this are noted in the individual
 `README.md` files.
 
+## OpenTelemetry .NET and relation with .NET Activity API
```
Comment (reyang) [inline README.md:22]: For folks who don't need Activity API (e.g. developers who just need to configure/enable telemetry) this part is less important. Consider moving this to the tracing part.

File: docs/metrics/getting-started.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ Metrics API and SDK is in alpha/experimental stage, and are not recommended for
 any production use. Both the API, and SDK implementation is subject to
 significant changes.
 
+Please check the [Metric support
+plan](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1501)._
```
Comment (reyang) [inline docs/metrics/getting-started.md:10]:
```suggestion
plan](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1501).
```

**Review by reyang — APPROVED** [review]
LGTM.

File: README.md [hunk 3]
```diff
@@ -21,8 +21,10 @@ Framework](https://dotnet.microsoft.com/download/dotnet-framework) except for
 
 ## Getting Started
 
-If you are new here, please see [get started in 5
-minutes](./docs/trace/getting-started/README.md).
+If you are new here, please see getting started doc for
+[logs](./docs/logs/getting-started/README.md),
+[metrics](./docs/metrics/getting-started.md) and
+[trace](./docs/trace/getting-started/README.md).
```
Comment (CodeBlanch) [inline README.md:27]:
```suggestion
If you are new here, please see getting started docs for
[logs](./docs/logs/getting-started/README.md),
[metrics](./docs/metrics/getting-started.md), and
[trace](./docs/trace/getting-started/README.md).
```

Comment (CodeBlanch) [inline README.md:27]: `logs, metrics, and trace` seems like it should be `logs, metrics, and tracing` but not sure if you are matching up to spec or something?

**Review by CodeBlanch — APPROVED** [review]
Sent a suggestion but LGTM

File: README.md (hunk 3 above)
Comment (cijothomas) [reply README.md]: Our namespaces and folder names are all `Trace` not tracing, so I sticked with that.

**Pull request merged**